test/test.js

```diff
@@ -0,0 +1,60 @@
+let assert = require("assert");
+let arrLib = require("../arr-lib");
+
+describe("arr-lib", function () {
+    describe("function max()", function () {
+        it("should return 5 when value is [1, 3, 2, 5, 3]", function () {
+            assert.strictEqual(arrLib.max([1, 3, 2, 5, 3]), 5);
+        });
+    });
+    describe("function min()", function () {
+        it("should return 1 when value is [1, 3, 2, 5, 3]", function () {
+            assert.strictEqual(arrLib.min([1, 3, 2, 5, 3]), 1);
+        });
+    });
+    describe("function mean()", function () {
+        it("should return 1 when value is [1, 3, 2, 5, 3]", function () {
+            assert.strictEqual(arrLib.mean([1, 3, 2, 5, 3]), 2.8);
+        });
+    });
+    describe("function sort()", function () {
+        it("should return [1, 2, 3, 3, 5] when value is [1, 3, 2, 5, 3] and 'bubble'", function () {
+            assert.deepStrictEqual(arrLib.sort([1, 3, 2, 5, 3],"bubble"), [1, 2, 3, 3, 5]);
+        });
+        it("should return [] when value is [] and 'bubble'", function () {
+            assert.deepStrictEqual(arrLib.sort([],"bubble"), []);
+        });
+        it("should return [1, 2, 3, 3, 5] when value is [1, 3, 2, 5, 3] and 'selection'", function () {
+            assert.deepStrictEqual(arrLib.sort([1, 3, 2, 5, 3],"selection"), [1, 2, 3, 3, 5]);
+        });
+    });
+    describe("function getMaxSubSum()", function () {
+        it("should return 14 when value is [1, 3, 2, 5, 3]", function () {
+            assert.deepStrictEqual(arrLib.getMaxSubSum([1, 3, 2, 5, 3]), 14);
```

Comment:
Deep comparing here and below is not needed

Reply:
I can't use just assert.equal() on arrays because an array is a reference type in JavaScript, and hence only the reference is compared but I can avoid strict comparison